R/checks.R

```diff
@@ -848,4 +848,48 @@ check_fracwith <- function(form, rep) {
                                & num_of_fractures != 1)))
   
   return(problems)
+}
+
+#' Check that the time from injury to def. stabilization should be within
+#'  1 hour of the time from injury to abx administration
+#' 
+#' @param form dataframe containing ptstatus, form 4.1, and form5.3
+#' @return a dataframe containing problematic entries with relevant columns
+#' @import tidyverse
+#' @export 
+check_ihhrs_ishrs_iahrs_1hrwithin <- function(form) {
+  hrswithin <- 1
+  problems <- form %>% 
+    transmute(
+      region, site, studyid, ptinit, ptstatus, 
+      ihhrstotal = if_else(!is.na(ihhrs), ihhrs, ihdays * 24),
```

Comment:
Please don't rely on is.na to check for units. There is a special column called ihunits.
This is true for all hours and days column. Refer to coding table for reference.